make-prs.py

```diff
@@ -0,0 +1,251 @@
+import argparse
+import logging
+import os
+import re
+import shutil
+import subprocess
+import typing
+
+import github
+import requests
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+WORKING_DIR = "./tmp"
```

Comment:
Does this have to be inside of the current directory structure? It seems nicer to use a `tempfile` temporary directory, if that's possible.

Reply:
When debugging, I find it helpful to have it in the current working directory which also shows up in the IDE, because you need to view the content of the clone quite frequently. I'm keeping it as it is.